src/wp-admin/edit-tags.php

```diff
@@ -348,9 +348,15 @@
 
 <hr class="wp-header-end">
 
-<?php if ( $message ) : ?>
-<div id="message" class="<?php echo $class; ?> notice is-dismissible"><p><?php echo $message; ?></p></div>
-	<?php
+<?php
+if ( $message ) :
+	wp_admin_notice(
+		$message,
+		array(
+			'additional_classes' => array( $class ),
```

Comment:
```suggestion
			'id'                 => 'message',
			'additional_classes' => array( $class ),
```
The div previously had an ID of `'message'`.